com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingMain.raytrace

```diff
@@ -53,6 +57,13 @@ void MissMaterial(inout PathIntersection pathIntersection : SV_RayPayload)
     ApplyFogAttenuation(WorldRayOrigin(), WorldRayDirection(), pathIntersection.value);
 }
 
+float2 UniformPointWithinCircle(float radius, float u1, float u2)
```

Comment:
We already have this in Sampling.hlsl  called SampleDiskUniform (need to multiply by radius outside)